examples/tv-casting-app/APIs.md

```diff
@@ -733,59 +842,166 @@ void ConnectionHandler(CHIP_ERROR err, matter::casting::core::CastingPlayer * ca
+`CastingPlayer` object it wants to connect to. Alternately, the Casting Client
+can connect to a `CastingPlayer`, using the `CastingPlayer` /
+Commissioner-Generated Passcode UDC feature by successfully calling
+`verifyOrEstablishConnection`, updating the commissioning passcode and then
+calling `continueConnecting` on the `CastingPlayer`.
 
 ```java
-private static final long MIN_CONNECTION_TIMEOUT_SEC = 3 * 60;
-
-EndpointFilter desiredEndpointFilter = new EndpointFilter();
-desiredEndpointFilter.vendorId = DESIRED_ENDPOINT_VENDOR_ID;
-
-MatterError err = targetCastingPlayer.verifyOrEstablishConnection(
-                      MIN_CONNECTION_TIMEOUT_SEC,
-                      desiredEndpointFilter,
-                      new MatterCallback<Void>() {
-                        @Override
-                        public void handle(Void v) {
-                          Log.i(
-                              TAG,
-                              "Connected to CastingPlayer with deviceId: "
-                                  + targetCastingPlayer.getDeviceId());
-                          getActivity()
-                              .runOnUiThread(
-                                  () -> {
-                                    connectionFragmentStatusTextView.setText(
-                                        "Connected to Casting Player with device name: "
-                                            + targetCastingPlayer.getDeviceName()
-                                            + "\n\n");
-                                    connectionFragmentNextButton.setEnabled(true);
-                                  });
-                        }
-                      },
-                      new MatterCallback<MatterError>() {
-                        @Override
-                        public void handle(MatterError err) {
-                          Log.e(TAG, "CastingPLayer connection failed: " + err);
-                          getActivity()
-                              .runOnUiThread(
-                                  () -> {
-                                    connectionFragmentStatusTextView.setText(
-                                        "Casting Player connection failed due to: " + err + "\n\n");
-                                  });
+private static final short MIN_CONNECTION_TIMEOUT_SEC = 3 * 60;
+private static final Integer DESIRED_TARGET_APP_VENDOR_ID = 65521;
+
+// Specify the TargetApp that the client wants to interact with after commissioning. If this value is passed in,
+// VerifyOrEstablishConnection() will force UDC, in case the desired TargetApp is not found in the on-device
+// CastingStore
+IdentificationDeclarationOptions idOptions = new IdentificationDeclarationOptions();
+TargetAppInfo targetAppInfo = new TargetAppInfo(DESIRED_TARGET_APP_VENDOR_ID);
+idOptions.addTargetAppInfo(targetAppInfo);
+
+// If using the alternate CastingPlayer / Commissioner-Generated Passcode UDC feature commissioning flow.
```

Comment:
editorial: The C in UDC is "commissioning", so omit the extra "commissioning", here and in the 7 other places.

```suggestion
// If using the alternate CastingPlayer / Commissioner-Generated Passcode UDC feature.
```